### PR TITLE
Add Logstash vscode plugins

### PR DESCRIPTION
**Pull request by disaster37**
Signed-off-by: disaster37 <linuxworkgroup@hotmail.com>

### What does this PR do?

Add 2 Logstash vsocde plugins:
  - https://open-vsx.org/extension/fbaligand/vscode-logstash-editor
  - https://open-vsx.org/extension/RandomChance/logstash

### What issues does this PR fix or reference?

It's an enhance

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: che-theia-plugins.yaml [hunk 1]
```diff
@@ -768,3 +768,11 @@ plugins:
     repository:
       url: 'https://github.com/BroadcomMFD/cobol-control-flow'
     extension: https://open-vsx.org/api/BroadcomMFD/ccf/0.4.0/file/BroadcomMFD.ccf-0.4.0.vsix
+  - repository:
+      url: 'https://github.com/randomchance/vscode-logstash-configuration-syntax'
+      revision: '0.0.4'
```
Comment (svor) [inline che-theia-plugins.yaml:773]: I don't see 0.0.4 tag in https://github.com/randomchance/vscode-logstash-configuration-syntax. Please see what revision attribute should describe: https://github.com/eclipse-che/che-plugin-registry/blob/main/CONTRIBUTE.md

File: che-theia-plugins.yaml [hunk 2]
```diff
@@ -768,3 +768,11 @@ plugins:
...
+    extension: 'https://open-vsx.org/api/RandomChance/logstash/0.0.4/file/RandomChance.logstash-0.0.4.vsix'
+  - repository:
+      url: 'https://github.com/fbaligand/vscode-logstash-editor'
+      revision: '1.1.3'
```
Comment (svor) [inline che-theia-plugins.yaml:777]: Probably it should be `v1.1.3` https://github.com/fbaligand/vscode-logstash-editor/tree/v1.1.3

**Review by svor — CHANGES_REQUESTED** [review]

File: che-theia-plugins.yaml (hunk 1 above)
Comment (disaster37) [reply che-theia-plugins.yaml]: You are right. I change revision by commit number

File: che-theia-plugins.yaml (hunk 2 above)
Comment (disaster37) [reply che-theia-plugins.yaml]: You are right, it's `v1.1.3` instead `1.1.3`

**Comment by svor** [comment]
Please rebase your changes to fix a conflict

**Review by svor — APPROVED** [review]

**Comment by themr0c** [comment]
Can someone add the label `status/release-notes-review-done` and remove the label `status/release-notes-review-needed`. I don't have sufficient rights.

**Pull request merged**